bin/trigger_azure_pipelines.py

```diff
@@ -10,12 +10,13 @@
 
 from azure.devops.connection import Connection
 from azure.devops.v6_0.pipelines.pipelines_client import PipelinesClient
-from azure.devops.v6_0.pipelines.models import Run
+from azure.devops.v6_0.build.build_client import BuildClient
+from azure.devops.v6_0.build.models import TimelineRecord
 from msrest.authentication import BasicAuthentication
 from requests import Response
 
 # https://learn.microsoft.com/en-us/rest/api/azure/devops/pipelines/runs/get?view=azure-devops-rest-6.0#runresult
-FAILED_STATUSES: set[str] = {"canceled", "failed", "unknown"}
+FAILED_STATUSES: set[str] = {"canceled", "failed", "skipped", "unknown"}

```

Comment:
There is another status called `abandoned`. And `unknown` status doesn't exist. https://learn.microsoft.com/en-us/rest/api/azure/devops/build/timeline/get?view=azure-devops-rest-6.0#taskresult. I think we should handle that scenario as well.